grails-app/conf/BuildConfig.groovy

```diff
@@ -30,16 +30,18 @@ grails.project.dependency.resolution = {
     repositories {
         inherits true
 
+        mavenRepo 'https://jenkins.aodn.org.au/plugin/repository/everything', {
+            updatePolicy 'always'
```

Comment:
This can also be set to 'daily' or any number of minutes (e.g. 18).

I'm not sure if having the frequency too high will cause any problem (particularly if Jenkins is down or restarted for any reason).